tools/debug-ui/scripts/compare_tries.py

```diff
@@ -0,0 +1,189 @@
+'''Performs diff of two tries between two nodes, by querying their Entity Debug API.
+This is useful when a node is stuck at InvalidStateRoot to debug why.'''
+import json
+import requests
+
+USE_COLD_STORAGE = False
```

Comment:
not used?

Reply:
Oh yeah, that was left over from an earlier script. I removed it. Ideally the cold storage flag would be used only on the non-stuck node but I can't really test that right now so I'll just remove it. It can be easily added later.